packages/diagrams/frontend/sirius-components-diagrams/src/renderer/node/FreeFormNode.tsx

```diff
@@ -58,6 +58,11 @@ const imageNodeStyle = (
     height: '100%',
     transform: rotation,
     background: getCSSColor(String(style.background), theme),
+    borderRadius: style.borderRadius,
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    zIndex: -1,
```

Comment:
Is this related to this PR?

Reply:
Yes, the background need to be in absolute position now